### ISSUE TITLE
Can you provide the train graphs from tensorboard?

### ISSUE THREAD

**Issue by sdahan12** [elvisyjlin/AttGAN-PyTorch#29]
Thanks for this great code!

**Comment by elvisyjlin** [comment]
Not at all. However, I'm sorry that I removed all the stuffs of AttGAN from my local machine since I graduated. You'll have to re-train the model from scratch to get the graph.
